packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -137,12 +193,12 @@ export class StorageService extends EventEmitter {
     await this.createDbForChannels()
     this.logger('2/6')
     await this.createDbForCertificates()
-    this.logger('3/6')
-    await this.createDbForMessageThreads()
+    // this.logger('3/6')
```

Comment:
If this is no longer needed, it should be removed. If you plan to bring it back in future please add a line of comment explaining why this is commented out